### PR TITLE
Randomize function added

### PR DESCRIPTION
**Pull request by Zionert**

### CODE REVIEW

File: src/apps/selectPartAndGroup/SelectPartAndGroup.jsx [hunk 1]
```diff
@@ -1,38 +1,73 @@
...
 
+import './SelectPartAndGroup.scss'
+
 const SelectPartAndGroup = () => {
 
   const { theme } = useContext(ThemeContext)
+  const [members, setMembers] = useState([])
   const [selectPart, setSelectPart] = useState(0)
   const [selectGroup, setSelectGroup] = useState(0)
+  const [buttonPress, setButtonPress] = useState(0)
```
Comment (ArtemKozar) [inline src/apps/selectPartAndGroup/SelectPartAndGroup.jsx]: Change value from 0 to false

File: src/apps/selectPartAndGroup/SelectPartAndGroup.jsx [hunk 2]
```diff
@@ -1,38 +1,73 @@
...
+      if (members) {
+        const copyArray = membersList;
+        for (let i = copyArray.length - 1; i > 0; i--) {
+          let j = Math.floor(Math.random() * (i + 1));
+          let temp = copyArray[i];
+          copyArray[i] = copyArray[j];
+          copyArray[j] = temp;
+      }
+        setMembers(copyArray) 
+        setButtonPress(buttonPress + 1)
```
Comment (ArtemKozar) [inline src/apps/selectPartAndGroup/SelectPartAndGroup.jsx]: replace buttonPress + 1 to !buttonPress

**Review by ArtemKozar — CHANGES_REQUESTED** [review]

File: src/apps/selectPartAndGroup/SelectPartAndGroup.jsx (hunk 1 above)
Comment (Zionert) [reply src/apps/selectPartAndGroup/SelectPartAndGroup.jsx]: Fixed comments

File: src/apps/selectPartAndGroup/SelectPartAndGroup.jsx (hunk 2 above)
Comment (Zionert) [reply src/apps/selectPartAndGroup/SelectPartAndGroup.jsx]: Fixed comments

**Pull request merged**